fairlogger.sh

```
@@ -6,10 +6,15 @@ requires:
  - CMake
- - "GCC-Toolchain:(?!osx)"
+ - GCC-Toolchain
+ - alibuild-recipe-tools
 incremental_recipe: |
   cmake --build . --target install ${JOBS:+-- -j$JOBS}
-  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+  mkdir -p "$INSTALLROOT/etc/modulefiles"
+  alibuild-generate-module --lib > "$INSTALLROOT/etc/modulefiles/$PKGNAME"
+  cat >> "$INSTALLROOT/etc/modulefiles/$PKGNAME" <<\EoF
+  prepend-path ROOT_INCLUDE_PATH $PKG_ROOT/include
+  EoF
```

Shellcheck (0.11.0)

[error] Remove indentation before end token (or use <<- and indent with tabs).

(SC1039)